src/components/qcResultsUpload/QcResultsUploadForm.vue

```diff
@@ -21,11 +21,24 @@
           required
         ></traction-file>
       </traction-sub-section>
-      <traction-button id="upload-button" type="submit" theme="create" size="lg" :disabled="busy">
+      <traction-button
+        id="upload-button"
+        type="submit"
+        theme="create"
+        size="lg"
+        :disabled="disableUpload"
```

Comment:
I dont know the background of the issue but can you use the busy variable instead of disableUpload? i.e disabled (busy) whilst the file is processing and enabled (not busy) after processing.